src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -592,7 +886,7 @@ private BoundDecisionDag MakeDecisionDag(SyntaxNode syntax, ImmutableArray<Parti
 
         private static ImmutableArray<PartialCaseDecision> RemoveEvaluation(ImmutableArray<PartialCaseDecision> cases, BoundDagEvaluation e)
         {
-            return cases.SelectAsArray(c => RemoveEvaluation(c, e));
+            return cases.SelectAsArray((c, eval) => RemoveEvaluation(c, eval), e);
         }
 
         private static PartialCaseDecision RemoveEvaluation(PartialCaseDecision c, BoundDagEvaluation e)
```

Comment:
Consider inlining this method.

Reply:
Resolved in [https://github.com/dotnet/roslyn/pull/25662](https://github.com/dotnet/roslyn/pull/25662)

---
In reply to: [176221170](https://github.com/dotnet/roslyn/pull/24059#discussion_r176221170) [](ancestors = 176221170)